cmd/ci-operator-prowgen/main_test.go

```diff
@@ -35,7 +35,7 @@ var unexportedFields = []cmp.Option{
 }
 
 func TestGeneratePodSpec(t *testing.T) {
-	testSecret := &ciop.Secret{Name: "test-secret", MountPath: "/usr/local/test-secret"}
+	testSecret := &ciop.Secret{Name: "secret-name", MountPath: "/usr/local/test-secret"}
```

Comment:
Why do we need this var?

Reply:
It was already there. It's not really necessary. Removed.